src/workerd/io/BUILD.bazel

```diff
@@ -80,6 +80,16 @@ wd_cc_library(
         ":set_enable_experimental_webgpu": ["//src/workerd/api/gpu:hdrs"],
         "//conditions:default": [],
     }),
+    # global CPU-specific options are inconvenient to specify with bazel – just set the options we
+    # need for CRC32C in this target for now.
+    copts = select({
```

Comment:
Note that we already use these options for building Chromium zlib, requirements are documented in README.